BrightID/src/components/Tasks/BituVerificationScreen.tsx

```diff
@@ -0,0 +1,222 @@
+import React, { useCallback } from 'react';
+import { StyleSheet, View, Text, Image, Linking, FlatList, ScrollView } from 'react-native';
+import { useDispatch, store, useSelector } from '@/store';
+import { useHeaderHeight } from '@react-navigation/stack';
+import { DEVICE_LARGE, DEVICE_IOS } from '@/utils/deviceConstants';
+import { photoDirectory } from '@/utils/filesystem';
+import { GREY, WHITE, BLUE, BLACK, RED } from '@/theme/colors';
+import { useTranslation } from 'react-i18next';
+import { fontSize } from '@/theme/fonts';
+import { selectAllConnections } from '@/reducer/connectionsSlice';
+
+export const BituVerificationScreen = function () {
+  const dispatch = useDispatch();
+  const { t } = useTranslation();
+  let headerHeight = useHeaderHeight();
+  if (DEVICE_IOS && DEVICE_LARGE) {
+    headerHeight += 7;
+  }
+  const {
+    user: { verifications },
+  } = store.getState();
+  const connections = useSelector(selectAllConnections);
+  const bituVerification = verifications.find(v => v.name == 'Bitu');
+  const directReports = bituVerification ?  Object.keys(bituVerification.directReports) : [];
+  const indirectReports = bituVerification ?  Object.keys(bituVerification.indirectReports) : [];
+  const readMoreUrl = 'https://brightid.gitbook.io/brightid/setting-up-social-recovery';
```

Comment:
Instead of local url definition we should use `url` from the `Bitu` entry in UserTasks.ts. Maybe passed in via route parameter.

Reply:
This is also a dummy link and we should update it.
I'm finalizing new gitbook section for Bitu and update the link too.